vars/commonPipelineEnvironment.groovy

```diff
@@ -21,6 +21,8 @@ class commonPipelineEnvironment implements Serializable {
 
     private String mtarFilePath
 
+    private String transportRequestId
```

Comment:
The reset should reset the transport id too.

Reply:
yes, you are right. I will change that.

Reply:
fixed